src/Illuminate/Support/ViewErrorBag.php

```diff
@@ -33,7 +33,7 @@ public function hasBag($key = 'default')
      */
     public function getBag($key)
     {
-        return Arr::get($this->bags, $key, new MessageBag);
+        return Arr::get($this->bags, $key) ?: new MessageBag;
```

Comment:
I agree. Nevertheless we do use this default value fallback everywhere in the framework, would it make sense to change all calls to `Arr::get` for this reason ? I don't think so